src/styles/block/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 7 * $day__size + $month__paddings + $gap__days * 6;
+  padding: $month__paddings;
+  gap: $gap__days;
+  
+  &__day {
+    box-sizing: border-box;
+    height: $day__size;
+    width: $day__size;
+    background-color: #eee;
+    transition-duration: 500ms;
+    border: $border__days solid black;
+    cursor: pointer;
+  
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.2s linear;
+    }
```

Comment:
It is recommended to write all `transition` parameters under general selector (not `:hover`). In this case transition will work both ways and the same.